### PR TITLE
[Paddle-TRT] Stack op plugin

### PR DESCRIPTION
**Pull request by zlsh80826**
### PR types

New features
### PR changes

OPs
### Describe

Add stack op converter

### CODE REVIEW

**Comment by jeng1220** [comment]
@NHZlX plz take a look

File: paddle/fluid/inference/tensorrt/convert/scale_op.cc [hunk 1]
```diff
@@ -73,6 +90,16 @@ class ScaleOpConverter : public OpConverter {
           power_weights.get(), scale_weights.get(), power_weights.get());
     }
 
+    PADDLE_ENFORCE_EQ(layer != nullptr, true);
```
Comment (jeng1220) [inline paddle/fluid/inference/tensorrt/convert/scale_op.cc]: @zlsh80826 ,
You have to add error message to pass CI-test:

CI-test log:
```sh
2020-08-05 00:49:22 0. The error message you wrote in PADDLE_ENFORCE{_**} or PADDLE_THROW does not meet our error message writing specification. Possible errors include 1. the error message is empty / 2. the error message is too short / 3. the error type is not specified. Please read the specification [ https://github.com/PaddlePaddle/Paddle/wiki/Paddle-Error-Message-Writing-Specification ], then refine the error message. If it is a mismatch, please request chenwhql (Recommend), luotao1 or lanxianghit review and approve.
2020-08-05 00:49:22 The PADDLE_ENFORCE{_**} or PADDLE_THROW entries that do not meet the specification are as follows:
2020-08-05 00:49:22 PADDLE_ENFORCE_EQ(layer != nullptr, true);
```

**Comment by jeng1220** [comment]
@NHZlX ,
What did this `PR-CI-CPU-Py2` message mean? How can we solve this issue?
```sh
2020-08-05 00:49:22 There are 1 approved errors.
2020-08-05 00:49:22 ****************
2020-08-05 00:49:23 + approval_error=1
2020-08-05 00:49:23 + '[' 1 '!=' 0 ']'
2020-08-05 00:49:23 + exit 6
2020-08-05 00:49:23 + :
2020-08-05 00:49:23 + EXCODE=6
2020-08-05 00:49:23 + echo 'EXCODE: 6'
2020-08-05 00:49:23 EXCODE: 6
2020-08-05 00:49:23 + '[' 6 -eq 0 ']'
2020-08-05 00:49:23 + exit 6
2020-08-05 00:49:23 {build code state=6}
2020-08-05 00:49:23 the build(4fb2de825288414187ee82897e65b6c7) state is BUILD_CODE_FAIL
```
Also this one, the message from `PR-CI-Coverage`:
```
2020-08-05 02:13:36 4.0K	/tmp/upload-265f9230-38fe-4045-8060-0c30af230445.tar.gz
2020-08-05 02:13:38 + export http_proxy=http://172.19.56.199:3128
2020-08-05 02:13:38 + http_proxy=http://172.19.56.199:3128
2020-08-05 02:13:38 + echo 'report uploaded'
2020-08-05 02:13:38 report uploaded
2020-08-05 02:13:38 + '[' 0 -eq 0 ']'
2020-08-05 02:13:38 + '[' ON == ON ']'
2020-08-05 02:13:38 + '[' '' == ON ']'
2020-08-05 02:13:38 + '[' 9 -eq 9 ']'
2020-08-05 02:13:38 + echo 'Coverage Failed!'
2020-08-05 02:13:38 Coverage Failed!
2020-08-05 02:13:38 + echo 9
2020-08-05 02:13:38 9
2020-08-05 02:13:38 + exit 9
2020-08-05 02:13:38 {build code state=9}
2020-08-05 02:13:38 the build(5bc5db7d76844035925646106f1eb316) state is BUILD_CODE_FAIL
```

File: paddle/fluid/inference/tensorrt/convert/scale_op.cc (hunk 1 above)
Comment (zlsh80826) [reply paddle/fluid/inference/tensorrt/convert/scale_op.cc]: I have added it in commit b750d6e

**Comment by jeng1220** [comment]
@NHZlX , @cryoco,
I checked the `PR-CI-Coverage`. The failure was caused by `test_hdfs`, which is nothing to do with stack plugin. Could we merge this PR now?

```
2020-08-09 23:06:17 W0809 15:06:16.926489  2491 shell.cc:410]  _shell_execute_cmd execute cmd:/usr/local/hadoop-2.7.7//bin/hadoop fs -mv hdfs_test_timeout new_hdfs_test_timeout ElapsedMS:8395.48, err_no:0 status:1
2020-08-09 23:06:17 test_hdfs failed
2020-08-09 23:06:17  ..E.E.
2020-08-09 23:06:17 ======================================================================
2020-08-09 23:06:17 ERROR: test_hdfs (test_hdfs.FSTest)
2020-08-09 23:06:17 ----------------------------------------------------------------------
2020-08-09 23:06:17 Traceback (most recent call last):
2020-08-09 23:06:17   File "/paddle/build/python/paddle/fluid/tests/unittests/test_hdfs.py", line 212, in test_hdfs
2020-08-09 23:06:17     self._test_touch(fs)
2020-08-09 23:06:17   File "/paddle/build/python/paddle/fluid/tests/unittests/test_hdfs.py", line 301, in _test_touch
2020-08-09 23:06:17     fs.touch(path, exist_ok=True)
2020-08-09 23:06:17 AttributeError: 'HDFSClient' object has no attribute 'touch'
2020-08-09 23:06:17 
2020-08-09 23:06:17 ======================================================================
2020-08-09 23:06:17 ERROR: test_local (test_hdfs.FSTest)
2020-08-09 23:06:17 ----------------------------------------------------------------------
2020-08-09 23:06:17 Traceback (most recent call last):
2020-08-09 23:06:17   File "/paddle/build/python/paddle/fluid/tests/unittests/test_hdfs.py", line 225, in test_local
2020-08-09 23:06:17     self._test_touch(fs)
2020-08-09 23:06:17   File "/paddle/build/python/paddle/fluid/tests/unittests/test_hdfs.py", line 301, in _test_touch
2020-08-09 23:06:17     fs.touch(path, exist_ok=True)
2020-08-09 23:06:17 TypeError: touch() got an unexpected keyword argument 'exist_ok'
2020-08-09 23:06:17 
2020-08-09 23:06:17 ----------------------------------------------------------------------
2020-08-09 23:06:17 Ran 6 tests in 69.799s
2020-08-09 23:06:17 
2020-08-09 23:06:17 FAILED (errors=2)
```

**Comment by zlsh80826** [comment]
@NHZlX, @cryoco, please take a look. Our follow-up optimization depends on this pr.

**Review by cryoco — APPROVED** [review]
LGTM

**Pull request merged**